provider/pkg/await/await.go

```diff
@@ -199,7 +201,8 @@ func Creation(c CreateConfig) (*unstructured.Unstructured, error) {
 					c.Context, c.Inputs.GetName(), types.ApplyPatchType, objYAML, options)
 
 				if errors.IsConflict(err) {
-					err = fmt.Errorf(`use the "pulumi.com/patchForce" annotation if you want to overwrite the existing values: %w`, err)
+					err = fmt.Errorf("SSA field conflict detected. see %s for troubleshooting help\n: %w",
```

Comment:
```suggestion
					err = fmt.Errorf("server-side apply field conflict detected. see %s for troubleshooting help\n: %w",
```

Spelling this out avoids a TLA, at the cost of a few extra characters.